Graph index rework (#737)
Macrometacorp/docs: docs/graphs/index.md

sidebar_position: 10
---

Macrometa graphs allow you to model, store, and query complex relationships between entities. In Macrometa GDN, you can leverage graphs to represent connected data as nodes (vertices) and edges.

Vertices represent the entities in the graph, while edges represent the relationships between these entities. By organizing data in this manner, you can efficiently perform complex traversals, pattern matching, and other graph-specific queries.

## High-Level Process

To get started with graphs in Macrometa GDN, follow these steps.

### Create Vertex Collections

> Vertex collections are used to store the vertices or nodes of your graph. Each vertex represents an entity in your data model, such as a person, a product, or an event.

Reviewer: we should probably mention that vertex collection is the same thing as document collection in the GUI.

Author: How's this?

> A vertex can be a document in a document collection or of an [edge collection](../collections/graph-edge/), so `edges` can be used as `vertices`. Which collections are used within a named graph is defined via _edge definitions_.

Reviewer 2: ```suggestion
A vertex can be a document in a document collection or an [edge collection](../collections/graph-edge/), so `edges` can be used as `vertices`. The _edge definitions_ define which collections are used within a named graph.
```
